### ISSUE TITLE
Combinatie btn btn-primary met extern of download

### ISSUE THREAD

**Issue by WiebeCnossen** [dso-toolkit/dso-toolkit#74]
Als een call-to-action button een externe link is, mag/moet je dan ook de class `extern` specificeren? Dus dan zou je `class="btn btn-primary extern"` krijgen. Het ziet er niet zo gek uit, maar is het de bedoeling? Dat wordt me uit de dso-toolkit site niet duidelijk.

Dezelfde vraag heb ik voor de combinatie van `btn btn-primary` met `download`.

**Comment by VolleMelk** [comment]
@MichielDHVD:

Visueel ziet dit er dan zo uit:

![image](https://user-images.githubusercontent.com/9696261/37288228-0f9311c6-2607-11e8-90ba-c9efbbc03e19.png)

![image](https://user-images.githubusercontent.com/9696261/37288262-1edbeb8a-2607-11e8-8e22-f667ad2caa52.png)

**Comment by MichielDHVD** [comment]
@WiebeCnossen  Dso-toolkit geeft hier inderdaad nog geen duidelijkheid over. 

Ik ga hier mee aan de slag en zal zelf ook meteen kijken of ik hier een soort update voor kan maken. 

@VolleMelk Dankjewel voor het toevoegen van een afbeelding.

**Comment by gofix** [comment]
Omdat nog niet expliciet gesteld is dat die NIET de bedoeling is, of dat het onwenselijk zou zijn, hebben we het in het CMS nu zo geïmplementeerd dat dit WEL is toegestaan. Als dit eventueel later moet wijzigen kan dat natuurlijk prima.

**Comment by MichielDHVD** [comment]
Dat is prima. Voorlopig zal deze optie blijven. Bij dit onderdeel gaat het meer om implementatie regels, wat wanneer gebruiken. Hier wordt momenteel naar gekeken.

**Comment by VolleMelk** [comment]
@MichielDHVD Wil je dat dit uiteen wordt gezet in de stijlgids? We kunnen deze modifiers op de buttons pagina (https://www.dso-toolkit.nl/unstable/components/detail/buttons.html) in een losse rij zetten. Wil je dit dan voor alle 3 de varianten? (primary, default en link?)

**Comment by MichielDHVD** [comment]
@VolleMelk Dat is goed en dan voor alle drie de varianten. 

Enigste opmerking: voor de primary het liefste een download icoontje, als voorbeeld voor aanvraag formulier downloaden (Contextueel) . In ieder geval geen externe link bij primary buttons. 

Voor default en link (Bedoel je hier mee tertiare button? ) maakt het het niet uit wat er achter komt te staan. 

Ik zal in de toekomst hier zelf nog een revisie op los laten. Er worden nog regels opgesteld over algemeen gebruik van button, iconen, componenten etc.

**Comment by MichielDHVD** [comment]
@VolleMelk Deze staat nog in specification. Is er nog iets nodig?

**Comment by VolleMelk** [comment]
Nee, je hebt gelijk: We gaan dit documenteren in de component library
